common/test/views/support/cleaner/CommercialMPUForFrontsTest.scala

```diff
@@ -0,0 +1,167 @@
+package views.support
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.scalatest.{FlatSpec, Matchers}
+
+import org.apache.commons.lang.StringEscapeUtils
+
+class CommercialMPUForFrontsTest extends FlatSpec with Matchers {
+
+/*
+ * The format we are using for the test data is treated as XML when toString()
+ * is run on it. To parse it into a JSoup element, it is necessary to remove
+ * all the XML character encodings that have been introduced.
+ */
+  private def parseTestData(doc: String): Document = {
+    Jsoup.parse(StringEscapeUtils.unescapeXml(doc))
+  }
+
+  private def clean(document: Document): Document = {
+    val cleaner = CommercialMPUForFronts(true)
+    cleaner.clean(document)
+    document
+  }
+
+  val frontWithThrasher = <html>
```

Comment:
It might be worth making a directory for the templates in `common/test/views/support/cleaner` and keeping them there. Up to suggestions if that doesn't make sense though